src/logic/FetchRemotes.ts

```diff
@@ -172,21 +172,14 @@ export function buildAppCardData(manifest: Manifest, user?: string, repo?: strin
 }
 
 export const fetchMonoManifest = async (type: RepoType) => {
-  // TODO: clean this up to use a lookup map or something...
-  if (type === "theme") {
-    const manifest = await fetch(THEMES_URL).then(res => res.json()).catch(() => null);
-    return manifest;
-  } else if (type === "extension") {
-    // const manifest = await fetch(EXTENSIONS_URL).then(res => res.json()).catch(() => null);
-    const manifest = extensionsManifest;
-    return manifest;
-  } else if (type === "app") {
-    // const manifest = await fetch(APPS_URL).then(res => res.json()).catch(() => null);
-    const manifest = appsManifest;
-    return manifest;
-  } else {
-    return [];
-  }
+  const manifest = {
+    theme: await fetch(THEMES_URL).then(res => res.json()).catch((err) => {console.error(err); return [];}),
```

Comment:
This will make the fetch call every time you run this function, not just for themes.